modules/ztp-image-based-upgrade.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Image-based upgrade for {sno} cluster with the {lcao}
+
+You can upgrade the platform version of a managed {sno} cluster with the {lcao}.
+Using a generated seed image allows you to skip up to two minor releases and it is faster than the standard upgrade method.
+
+You generate the seed image from a {sno} seed cluster, which is deployed and configured with the desired {product-title} and applications.
```

Comment:
maybe we would give a little bit more detail on what is a "seed cluster". We have a hub cluster, then we have any spokes cluster, and now a new concept "seed cluster". Which is out of the ZTP hub/spokes clusters. 
Then we can refer to this definition on other parts of the documentation.